jinja2/includes/google_analytics.html

```diff
@@ -1,6 +1,8 @@
-{% if config.GOOGLE_ANALYTICS_ACCOUNT != '0' %}
-{% set GA_FILE = 'analytics_debug' if settings.DEBUG else 'analytics' %}
 <script>
+    // Mozilla DNT Helper
```

Comment:
This is not a standard HTML comment(i.e. <!-- Mozilla DNT Helper --> but I guess Jinja is ok with that?

Reply:
hmm I just copied the style of ``// only load GA if DNT is not enabled`` from lower. Do you want both to change to comment blocks like ``/* Mozilla DNT Helper */``?

Reply:
Oh doi ;p I was looking at the file extension and not thinking that this is a script block :) Ignore my comment